applications/FluidDynamicsApplication/python_scripts/python_solvers_wrapper_fluid.py

```diff
@@ -43,7 +43,7 @@ def CreateSolverByParameters(model, solver_settings, parallelism):
     # Solvers for MPI parallelism
     elif (parallelism == "MPI"):
         if solver_type == "monolithic" or solver_type == "Monolithic":
-            solver_module_name = "trilinos_navier_stokes_solver_vmsmonolithic"
+            solver_module_name = "trilinos_navier_stokes_monolithic_solver"
 
         elif solver_type == "fractional_step" or solver_type == "FractionalStep":
             solver_module_name = "trilinos_navier_stokes_solver_fractionalstep"
```

Comment:
We have some inconsistent naming still... Are we gonna change those also?

Reply:
Yes. But I'd do so in a separate PR.